docs/reference/how-to/indexing-speed.asciidoc

```diff
@@ -81,13 +81,10 @@ ongoing basis when compared to dedicated local storage. If you put an index on
 `EBS`, be sure to use provisioned IOPS otherwise operations could be quickly
 throttled.
 
-Stripe your index across multiple SSDs by configuring a RAID 0 array. Remember
-that it will increase the risk of failure since the failure of any one SSD
-destroys the index. However this is typically the right tradeoff to make:
-optimize single shards for maximum performance, and then add replicas across
-different nodes so there's redundancy for any node failures. You can also use
-<<modules-snapshots,snapshot and restore>> to backup the index for further
-insurance.
+Use multiple I/O devices (by specifying multiple `path.data` paths) to hold
+the shards on your node to improve I/O performance. Note that an OS-level
```

Comment:
Personally I find RAID 0 to be fine for two disks. We had great success with it. We lost a disk, lost the node, the cluster healed quickly and it was no big deal. Such a big deal no one called me to tell me about it or anything. I found out an hour or two later. If you have the spares (disks and nodes) and the hands to replace the disks and rebuilding the system is fairly easy then RAID 0 of two disks is fine for Elasticsearch. Three if you are feeling giddy.

I think we should keep it documented as a tradeoff.